### DISCUSSION TITLE
how do i install this?

### DISCUSSION THREAD

**Discussion by PolskaKrowa** [Pycord-Development/pycord#450] [Q&A] [answered]
i have tried to install the program on windows but it just shows with "no module named pip"

- [✓]  I have used powershell
- [✓]  I have used command prompt

**Comment by Dorukyum** [comment, marked as answer]
Looks like your pip is not in PATH. Try `python -m pip` and `py -m pip`.

**Reply by PolskaKrowa** [reply]
now its saying that i don't have python installed

**Reply by PolskaKrowa** [reply]
had to install a new version of python but thanks
